From 25b381881543deda26b204fecc77f50e6076865d Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:09:26 -0400
Subject: [usb] cdc-acm: use BIT macro

Message-id: <1410887463-20674-495-git-send-email-dzickus@redhat.com>
Patchwork-id: 95666
O-Subject: [RHEL7 PATCH 494/591] USB: cdc-acm: use BIT macro
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit d1b781002b31247d38a2892c51b88348d6a8f201
Author: Oliver Neukum <oneukum@suse.de>
Date:   Mon May 19 13:55:25 2014 +0200

    USB: cdc-acm: use BIT macro

    Converting the header to BIT for readability. No functional
    change.

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/class/cdc-acm.h b/drivers/usb/class/cdc-acm.h
index 80826f8..fc75651 100644
--- a/drivers/usb/class/cdc-acm.h
+++ b/drivers/usb/class/cdc-acm.h
@@ -126,9 +126,9 @@ struct acm {
 #define CDC_DATA_INTERFACE_TYPE 0x0a
 
 /* constants describing various quirks and errors */
-#define NO_UNION_NORMAL   1
-#define SINGLE_RX_URB   2
-#define NO_CAP_LINE   4
-#define NOT_A_MODEM   8
-#define NO_DATA_INTERFACE  16
-#define IGNORE_DEVICE   32
+#define NO_UNION_NORMAL   BIT(0)
+#define SINGLE_RX_URB   BIT(1)
+#define NO_CAP_LINE   BIT(2)
+#define NOT_A_MODEM   BIT(3)
+#define NO_DATA_INTERFACE  BIT(4)
+#define IGNORE_DEVICE   BIT(5)
-- 
1.7.1